Won't build on Mac OS X Monterey

```
Building with:
  Version string:  unknown
  Architecture:    x86_64
  DSP mix:         x86
  RTLSDR support:  yes
  BladeRF support: no
  HackRF support:  no
  LimeSDR support: no
cc  -I. -D_POSIX_C_SOURCE=200112L -DMODES_DUMP1090_VERSION=\"unknown\" -DMODES_DUMP1090_VARIANT=\"dump1090-fa\" -DMISSING_NANOSLEEP -DENABLE_CPUFEATURES -Icpu_features/include -DENABLE_RTLSDR -DSTARCH_MIX_X86 -O3 -g -std=c11 -fno-common -Wall -Wmissing-declarations -Werror -W -I/usr/local/Cellar/librtlsdr/0.6.0/include/ -I/usr/local/Cellar/libusb/1.0.24/include/libusb-1.0 -c net_io.c -o net_io.o
net_io.c:1086:15: error: implicit declaration of function 'alloca' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
    buf = p = alloca(len);
              ^
net_io.c:1086:13: error: incompatible integer to pointer conversion assigning to 'char *' from 'int' [-Werror,-Wint-conversion]
    buf = p = alloca(len);
            ^ ~~~~~~~~~~~
2 errors generated.
make: *** [net_io.o] Error 1
```

notes | 05/31/22 | 2:26AM | chris-dev 
/Users/chris-dev/Documents/dump1090:~$ make         
Building with:
  Version string:  unknown
  Architecture:    arm64
  DSP mix:         generic
  RTLSDR support:  no
  BladeRF support: no
  HackRF support:  no
  LimeSDR support: no
cc  -I. -D_POSIX_C_SOURCE=200112L -DMODES_DUMP1090_VERSION=\"unknown\" -DMODES_DUMP1090_VARIANT=\"dump1090-fa\" -DMISSING_NANOSLEEP -DSTARCH_MIX_GENERIC -O3 -g -std=c11 -fno-common -Wall -Wmissing-declarations -Werror -W -c dump1090.c -o dump1090.o
cc  -I. -D_POSIX_C_SOURCE=200112L -DMODES_DUMP1090_VERSION=\"unknown\" -DMODES_DUMP1090_VARIANT=\"dump1090-fa\" -DMISSING_NANOSLEEP -DSTARCH_MIX_GENERIC -O3 -g -std=c11 -fno-common -Wall -Wmissing-declarations -Werror -W -c anet.c -o anet.o
cc  -I. -D_POSIX_C_SOURCE=200112L -DMODES_DUMP1090_VERSION=\"unknown\" -DMODES_DUMP1090_VARIANT=\"dump1090-fa\" -DMISSING_NANOSLEEP -DSTARCH_MIX_GENERIC -O3 -g -std=c11 -fno-common -Wall -Wmissing-declarations -Werror -W -c interactive.c -o interactive.o
cc  -I. -D_POSIX_C_SOURCE=200112L -DMODES_DUMP1090_VERSION=\"unknown\" -DMODES_DUMP1090_VARIANT=\"dump1090-fa\" -DMISSING_NANOSLEEP -DSTARCH_MIX_GENERIC -O3 -g -std=c11 -fno-common -Wall -Wmissing-declarations -Werror -W -c mode_ac.c -o mode_ac.o
cc  -I. -D_POSIX_C_SOURCE=200112L -DMODES_DUMP1090_VERSION=\"unknown\" -DMODES_DUMP1090_VARIANT=\"dump1090-fa\" -DMISSING_NANOSLEEP -DSTARCH_MIX_GENERIC -O3 -g -std=c11 -fno-common -Wall -Wmissing-declarations -Werror -W -c mode_s.c -o mode_s.o
cc  -I. -D_POSIX_C_SOURCE=200112L -DMODES_DUMP1090_VERSION=\"unknown\" -DMODES_DUMP1090_VARIANT=\"dump1090-fa\" -DMISSING_NANOSLEEP -DSTARCH_MIX_GENERIC -O3 -g -std=c11 -fno-common -Wall -Wmissing-declarations -Werror -W -c comm_b.c -o comm_b.o
cc  -I. -D_POSIX_C_SOURCE=200112L -DMODES_DUMP1090_VERSION=\"unknown\" -DMODES_DUMP1090_VARIANT=\"dump1090-fa\" -DMISSING_NANOSLEEP -DSTARCH_MIX_GENERIC -O3 -g -std=c11 -fno-common -Wall -Wmissing-declarations -Werror -W -c net_io.c -o net_io.o
net_io.c:1086:15: error: implicit declaration of function 'alloca' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
    buf = p = alloca(len);
              ^
net_io.c:1086:13: error: incompatible integer to pointer conversion assigning to 'char *' from 'int' [-Werror,-Wint-conversion]
    buf = p = alloca(len);
            ^ ~~~~~~~~~~~
2 errors generated.
make: *** [net_io.o] Error 1

--- a/dump1090.h
+++ b/dump1090.h
@@ -79,6 +79,7 @@
 #include <sys/ioctl.h>
 #include <time.h>
 #include <limits.h>
+#include <strings.h>
 
 #include "compat/compat.h"
 #include "dsp/generated/starch.h"

--- a/net_io.c
+++ b/net_io.c
@@ -1083,7 +1083,11 @@ void sendBeastSettings(struct client *c, const char *settings)
     char *buf, *p;
 
     len = strlen(settings) * 3;
-    buf = p = alloca(len);
+    buf = p = malloc(len);
+    if (!buf) {
+        fprintf(stderr, "Out of memory sending beast settings string\n");
+        exit(1);
+    }
 
     while (*settings) {
         *p++ = 0x1a;
@@ -1092,6 +1096,7 @@ void sendBeastSettings(struct client *c, const char *settings)
     }
 
     anetWrite(c->fd, buf, len);
+    free(buf);
 }
 
 // Move a network client to a new service

